### PR TITLE
Adds the new health analyser noises to PDA scanners and mech scanners

### PR DESCRIPTION
**Pull request by Jaraci**
See title.

### CODE REVIEW

**Comment by Jaraci** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
You need to put in the spam limiter.

**Comment by Jaraci** [comment]
> You need to put in the spam limiter.

That should be it now, unless I'm mistaken!

File: code/modules/heavy_vehicle/equipment/medical.dm [hunk 1]
```diff
@@ -280,9 +280,12 @@
 		to_chat(user, SPAN_NOTICE("You switch to \the [src]'s [HA.fullScan ? "full body" : "basic"] scan mode."))
 
 /obj/item/device/healthanalyzer/mech/attack(mob/living/M, var/mob/living/heavy_vehicle/user)
+	if(last_scan <= world.time - 20) //Spam limiter.
```
Comment (alsoandanswer) [inline code/modules/heavy_vehicle/equipment/medical.dm:283]:
```suggestion
	sound_scan = FALSE
	if(last_scan <= world.time - 20) //Spam limiter.
```

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**